<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Бутерброд</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../res/ingredients/logo.png</normaloff>../res/ingredients/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="fridge_widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>651</width>
      <height>181</height>
     </rect>
    </property>
    <widget class="QWidget" name="cheese" native="true">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>99</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_12"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="bottom" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_14">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_13"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="salad" native="true">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tomatoes" native="true">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>99</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_8"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="onion" native="true">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>100</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_10"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="beef" native="true">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="cucumbers" native="true">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>10</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_6"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="top" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_15">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_16"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QListWidget" name="hamburgers_layers">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>340</y>
      <width>111</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
	border: 1px solid black;
	background-color: rgba(0, 0, 0, 0);
}</string>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::InternalMove</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1015</width>
      <height>700</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../res/othres/back_2.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>510</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>380</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Перетащите ингредиент в оласть слева, вы также моежете менять их местами</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>fridge_widget</zorder>
   <zorder>hamburgers_layers</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../res/main.qrc"/>
 </resources>
 <connections/>
</ui>
